modules/core/client/config/core-admin.client.routes.js

```diff
@@ -8,8 +8,13 @@ angular.module('core.admin.routes').config(['$stateProvider',
         abstract: true,
         url: '/admin',
         template: '<ui-view/>',
+        resolve: {
+          pageTitle: function () {
+            return 'MEAN.JS';
+          }
+        },
         data: {
-          roles: ['admin']
+          roles: ['admin'],
```

Comment:
Remove comma. There's a few other places that have unnecessary comma's after the `data.roles` definition.